src/harness/sas_interface.py

```diff
@@ -256,6 +256,18 @@ def PreloadRegistrationData(self, request):
     """
     pass
 
+  @abc.abstractmethod
+  def InjectExclusionZone(self, request, frequency_range):  
+    """Inject exclusion zone information into SAS under test.
+
+    Args:
+      request: A dictionary with a single key-value pair where the key is
+        "record" and the value is ZoneData object and frequency range to be injected into 
```

Comment:
This does not match the testing API doc.

Reply:
Modified as per InjectAPI definition